product_landing_page/index.css

```diff
@@ -159,6 +168,6 @@ body {
   }
 
   .advantages-section {
-    grid-column-gap: 1.5rem;
+    grid-column-gap: 1rem;
```

Comment:
Jest jakaś intencja za tą zmianą czy to przez przypadek? :P

Reply:
przypadek, już usuwam

Reply:
Przypadek :D